notebooks/official/generative_ai/text_embedding_api_cloud_next_new_models.ipynb

```diff
@@ -0,0 +1,363 @@
+{
```

Comment:
<p>Recommend to provide more information to help users to understand:</p><ol><li>Where to access a list of supported models</li><li>How to use the API</li><li>How to process the embeddings</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>